great_expectations/experimental/column_descriptive_metrics/cloud_data_store.py

```diff
@@ -0,0 +1,25 @@
+from typing import Union
+
+from typing_extensions import TypeAlias
+
+from great_expectations.data_context import CloudDataContext
+from great_expectations.experimental.column_descriptive_metrics.metrics import Metrics
+
+CloudStorableTypes: TypeAlias = Union[Metrics,]  # TODO: are there better approaches?
+
+
+class CloudDataStore:
+    def __init__(self, context: CloudDataContext):
+        # TODO: Get credentials from Cloud Data Context on init
+        self._context = context
+        pass
+
+    def create(
+        self, value_type: CloudStorableTypes, value: CloudStorableTypes
```

Comment:
these arguments should not be cloud specific

Reply:
consider introducing an abstract class `DataStore` that defines the contract (create, read, update, delete)

Reply:
Good call, both done 🙇